docker/lr-pb/shard_bam.py

```diff
@@ -0,0 +1,124 @@
+import argparse
+import gzip
+from collections import OrderedDict
+from math import ceil
+import pysam
+from construct import *
+
+from multiprocessing.pool import ThreadPool
+from functools import partial
+
+
+def write_shard(bam, offsets, prefix, index):
+    """
+    Write subsection of bam to a shard.
```

Comment:
can you make this docstring a bit more specific, i.e. 
   * it is assuming a pbi index, therefore is definitely NOT intended for ONT bam, or at least it has not been tested (integration test) that way

or have the equivalent for this whole script as opposed to this function.

Reply:
ping again

Reply:
Oops, missed this comment the first time.  I'll clarify the use for this function.

Reply:
Clarified that this applies to PacBio BAMs.